src/libs/migrations/MoveToIndexedDB.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+
+import Log from '../Log';
+import ONYXKEYS from '../../ONYXKEYS';
+import getPlatform from '../getPlatform';
+import CONST from '../../CONST';
+
+function shouldMigrate() {
+    const isTargetPlatform = _.contains([CONST.PLATFORM.WEB, CONST.PLATFORM.DESKTOP], getPlatform());
+    if (!isTargetPlatform) {
+        Log.info('[Migrate Onyx] Skipped migration MoveToIndexedDB (Not applicable to current platform)');
+        return false;
+    }
+
+    const session = window.localStorage.getItem(ONYXKEYS.SESSION);
+    if (!session || !session.includes('authToken')) {
+        Log.info('[Migrate Onyx] Skipped migration MoveToIndexedDB (Not applicable to logged out users)');
+        return false;
+    }
+
+    return true;
+}
+
+function migrate() {
+    const multiSetData = {};
+    const possibleKeys = _.union(_.values(ONYXKEYS), _.values(ONYXKEYS.COLLECTION));
+
+    for (let i = 0; i < window.localStorage.length; i += 1) {
+        const key = window.localStorage.key(i);
+        if (_.some(possibleKeys, entry => key.startsWith(entry))) {
```

Comment:
Can you please add a code comment to explain what this code is is doing and why?